lib/travis/api/v3/routes.rb

```diff
@@ -86,7 +86,7 @@ module Routes
     end
 
     resource :owner do
-      route '/owner/({owner.login}|{user.login}|{organization.login}|github_id/{owner.github_id})'
+      route '/owner/({owner.login}|{user.login}|{organization.login}|github_id/{owner.github_id}|github_installation_id/{owner.github_installation_id})'
```

Comment:
I think it would probably be more elegant to have installation be its own retrievable entity (ie `/installation/{installation.github_id}`), with a shorthand to fetch the owner directly (`/installation/{installation.github_id}/owner`), though that one would be optional, as the owner could also be eager-loaded (`/installation/{installation.github_id}?include=installation.owner`).